frontend/src/pages/Pictures.tsx

```diff
@@ -0,0 +1,51 @@
+import { createSignal, onMount } from "solid-js";
+import { useNavigate } from '@solidjs/router';
+import DefaultPicture from '../assets/images/newpicker.png';
+
+export default function Pictures() {
+    const navigate = useNavigate();
+    const [pictureList, setPictureList] = createSignal([DefaultPicture, DefaultPicture, DefaultPicture, DefaultPicture, DefaultPicture, DefaultPicture])
+
+    const handleImage = (index : number) => (e: any) => {
+        const image = e.target.files[0];
+        if (image) {
+            const reader = new FileReader();
+            reader.onload = (event) => {
+                const newPictureList = [...pictureList()];
+                if (event.target !== null) {
+                    newPictureList[index] = event.target.result as string;
+                    setPictureList(newPictureList);
+                } else {
+                    console.log("Unable to add image please try again.")
+                }
+            }
+            reader.readAsDataURL(image);
+        } else {
+            console.log("Couldn't get image please try again.")
+        }
+    }
+    return (
+        <div class="section-container gap-32">
+            <div class="flex items-center justify-center flex-col">
+                <p class="section-title">Please select photos to</p>
+                <p class="section-title">use for your profile.</p>
+            </div>
+            <div class="grid grid-cols-2 gap-4">
+                {pictureList().map((pic, index) => (
+                        <div key={index} class="relative">
+                            <input 
+                                type="file" 
+                                class="absolute inset-0 w-full h-full opacity-0 cursor-pointer" 
```

Comment:
The cursor-pointer only triggering on the bottom half is because the default "pick file" button is still there, you just can't see it:
![image](https://github.com/wellendorfmatthew/hana-solidjs/assets/157227076/dc9cda80-fbda-4512-8cbb-e9e672c0d804)
I was able to fix it by setting the z-index of the input element to `-z-10` and wrapping the image in a button with the `cursor-pointer` tag:
```js
  <input
              type="file"
              class="absolute inset-0 w-full h-full cursor-pointer -z-10 opacity-0"
              accept="image/*"
              onchange={handleImage(index)}
            />
            <button class="cursor-pointer">
              <img
                src={pic}
                alt=""
                class="w-[60px] h-[60px] object-cover pointer-events-none"
              />
            </button>
          </div>
```